source/reference/minio-mc-deprecated.rst

```diff
@@ -55,6 +51,10 @@ Table of Deprecated Commands
      - :mc-cmd:`mc ilm rule rm`
      - mc RELEASE.2022-12-24T15-21-38Z
 
+   * - ``mc replicate diff``
```

Comment:
There was a bunch of alphabetization fail in this file

Reply:
This, uh, was intentional: order by date deprecated. Discussing suitable ordering.

Reply:
Consensus says "meh alpha is ok." Can revisit if needed.